cypress/local-only/tests/integration/statusReportOrder/status-report-order-description-fields.cy.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  FORMATS,
+  formatNow,
+} from '../../../../../shared/src/business/utilities/DateHandler';
+import {
+  docketNumber,
+  getLastDraftOrderElementFromDrafts,
+} from '../../../support/statusReportOrder';
+import {
+  loginAsColvin,
+  loginAsColvinChambers,
+  loginAsDocketClerk,
+} from '../../../../helpers/authentication/login-as-helpers';
+import { logout } from '../../../../helpers/authentication/logout';
+
+describe('should default status report order descriptions', () => {
+  const today = formatNow(FORMATS.MMDDYYYY);
+  it('should display default description when document type is an Order', () => {
+    judgeOrChambersCreatesStatusReportOrder(today);
+    loginAsDocketClerk();
+    cy.visit(`/case-detail/${docketNumber}`);
+    cy.get('#tab-drafts').click();
+    getLastDraftOrderElementFromDrafts().click();
+    cy.get('[data-testid="add-court-issued-docket-entry-button"]').click();
+    cy.get('.select-react-element__control').should('have.text', 'Order');
```

Comment:
Let's try avoiding third-party library selectors. I think we can just do `#document-type` here.

Reply:
Same thing anyplace we use this selector below.